docs/rules/no-empty-glimmer-component-classes.md

```diff
@@ -26,6 +26,15 @@ import Component from '@glimmer/component';
 class MyComponent extends Component {}
 ```
 
+<!-- markdownlint-disable-next-line MD040 -->
+```
```

Comment:
Hopefully there is some way we can use can continue to use JS code highlighting in these markdown code samples. We can skip it for now if we need to, but there should be some way to configure ESLint to parse it correctly, given that ESLint can handle this syntax inside actual Ember apps.

Note that there is a section in our `.eslintrc.js` where we can customize the ESLint config for these markdown code samples:

https://github.com/ember-cli/eslint-plugin-ember/blob/a8efed70a4e0e1ce72580bc8b2d5647d200240dd/.eslintrc.js#L168-L171

Reply:
@bmish Inside actual Ember apps we process the result before running it through eslint so there is no parsing issue. Unfortunately doing an eslint disable still has a parsing issue since it doesn't know what to do with the `<template>` tag. An option is to ignore this md file and use `js` which produces syntax highlighting.

@chriskrycho do you have any thoughts or is supporting this in the roadmap? I can see that the [ember-template-imports](https://github.com/ember-template-imports/ember-template-imports) repo just uses `js` for markdown and doesn't run eslint it too.

Reply:
Good point. Yes I think telling ESLint to ignore it would be best then.

Reply:
Fix here https://github.com/ember-cli/eslint-plugin-ember/pull/1868